docs/bits/type/int.md

```diff
@@ -0,0 +1,11 @@
+# int
+
+defined by
+```int (<ranges list>)```
+
+If no range is provided, then represents infinite domain of integers.
+
+An integer range is either a single integer ```int(n)```, or a list of sequential integers with a given lower and upper bound 
+ ```int(l..u)```
+
+Values must be between -2<sup>62</sup> + 1 and 2<sup>62</sup> - 1
```

Comment:
Maybe also say that ranges can be combined using a comma. so something like `int(n,l..u,k..r)` is valid.